src/lib/ui/device/BatteryStatus.tsx

```diff
@@ -5,32 +5,27 @@ import { BatteryLevelFullIcon } from 'lib/icons/BatteryLevelFull.js'
 import { BatteryLevelHighIcon } from 'lib/icons/BatteryLevelHigh.js'
 import { BatteryLevelLowIcon } from 'lib/icons/BatteryLevelLow.js'
 
-export function BatteryStatus(props: {
-  device: LockDevice
-}): JSX.Element | null {
+export function BatteryStatus(props: { device: LockDevice }) {
```

Comment:
Prefer `export interface BatteryStatusProps` over inline for the main named component in the module.

Reply:
Updated in https://github.com/seamapi/react/pull/124/commits/72461f87302670be08f6e88bb5968294f9f50b0e